classes/app.rb

```diff
@@ -33,6 +35,8 @@ class App
   include PreserveMusicAlbumGenre
   include BookJson
   include LabelJson
+  include AuthorsJson
+  include GamesJson
 
   def initialize
```

Comment:
Kindly remove add_author from console as requirements for project doesn't need it and you have same already in create game